static/app/views/settings/components/settingsLayout.tsx

```diff
@@ -15,15 +15,15 @@ import SettingsSearch from './settingsSearch';
 
 type Props = {
   children: React.ReactNode;
-  renderNavigation?: () => React.ReactNode;
+  renderNavigation?: (opts: {mobileVisible: boolean}) => React.ReactNode;
 } & RouteComponentProps<{}, {}>;
 
 function SettingsLayout(props: Props) {
-  // This is used when the screen is small enough that the navigation should
-  // be hidden
+  // This is used when the screen is small enough that the navigation should be
+  // hidden. This state is only used when the media query matches.
   //
   // [!!] On large screens this state is totally unused!
-  const [navVisible, setNavVisible] = useState(false);
+  const [mobileVisible, setMobileNavVisible] = useState(false);
```

Comment:
Should this be `mobileNavVisible`?

Reply:
Yeah that would probably be a bit more clear

Reply:
Renamed to `isMobileNavVisibile`